tests/runtime/engine/runner.py

```diff
@@ -201,10 +201,13 @@ def run_test(test):
 
         def get_pid_ns_cmd(cmd):
             return nsenter + [os.path.abspath(x) for x in cmd.split()]
-            
+
         def check_expect(expect, output):
             try:
-                if expect.mode == "regex":
+                if expect.mode == "text":
+                    # Raw text match on an entire line, ignoring leading/trailing whitespace
+                    return re.search(f"^\s*{re.escape(expect.expect)}\s*$", output, re.M)
```

Comment:
Do we need a regex here at all if we're matching on an exact substring? Would a simple `return expect.expect in output` work?

Reply:
I did start with something a lot simpler before I ended up with this nasty regex :P

The reason for the regex is that it lets us perform a substring match but still ensure that it's anchored to the start and end of a line, to avoid a situation like
`EXPECT val = 1` matching the string `val = 10`